client/my-sites/upgrades/checkout-thank-you/domain-mapping-details.jsx

```diff
@@ -2,31 +2,40 @@
  * External dependencies
  */
 import React from 'react';
-import i18n from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
+import { localize } from 'i18n-calypso';
 import PurchaseDetail from 'components/purchase-detail';
 import supportUrls from 'lib/url/support';
 
-const DomainMappingDetails = ( { domain, registrarSupportUrl } ) => {
-	const registrarSupportLink = registrarSupportUrl ? <a target="_blank" rel="noopener noreferrer" href={ registrarSupportUrl } /> : <span />;
+const DomainMappingDetails = ( { domain, registrarSupportUrl, translate } ) => {
+	const registrarSupportLink = registrarSupportUrl
+		? <a target="_blank" rel="noopener noreferrer" href={ registrarSupportUrl } />
+		: <span />;
 	const description = (
 		<div>
 			<p>
 				{
-					i18n.translate( 'The domain {{em}}%(domain)s{{/em}} still has to be configured to work with WordPress.com.', {
+					translate( 'Your domain {{em}}%(domain)s{{/em}} has to be configured to work with WordPress.com.', {
 						args: { domain },
 						components: { em: <em /> }
 					} )
 				}
 			</p>
 			<p>
 				{
-					i18n.translate(
-						'You will need to log into {{registrarSupportLink}}your registrar\'s site{{/registrarSupportLink}} ' +
-						'and change the "Name Servers" to:',
+					translate(
+						'If you already did this yourself, or if the domain was already configured for you, no further action is needed.'
+					)
+				}
+			</p>
+			<p>
+				{
+					translate(
+						'If not, you will need to log into {{registrarSupportLink}}your registrar\'s site{{/registrarSupportLink}} ' +
+						'(where you purchased the domain originally) and change the "Name Servers" to:',
```

Comment:
Do we still need the message in parentheses if we already have a link to registrar?